keyboards/kbdfans/maja/maja.c

```diff
@@ -102,3 +102,12 @@ void suspend_wakeup_init_kb(void) {
     rgb_matrix_set_suspend_state(false);
     suspend_wakeup_init_user();
 }
+
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void)
```

Comment:
Is this in the keymap, as well? 

If not, I think it should be, just in case.

Reply:
it's not in the keymap.